contracts/serverless-v2/Executors.sol

```diff
@@ -130,12 +130,21 @@ contract Executors is
         _;
     }
 
+    modifier isValidEnv(uint8 _env) {
+        if(!isSupportedEnv[_env])
+            revert ExecutorsEnvUnsupported();
+        _;
+    }
+
+    mapping(uint8 => bool) public isSupportedEnv;
```

Comment:
maybe if it would be just ExecutionEnvs

Reply:
this seemed more intuitive as it represents a bool val